sections/combat.tex

```diff
@@ -71,7 +71,8 @@ \subsection*{\hypertarget{Combatterminology}{Combat Terminology}}
 That Unit \textbf{must skip its next Activation}. Remove the Token instead of activating it.
 If the Unit \textbf{attacked or takes any damage} before that time, \textbf{remove the Token}.
 The Token does not prevent Units from performing Retaliation Attacks.\par
-\textbf{\hypertarget{Defend}{Defend}} \includesvg[height=10px]{\svgs/defense.svg} – Units may choose to gain a Defense Token instead of attacking.
+\textbf{\hypertarget{Defend}{Defend}} \includesvg[height=10px]{\svgs/defense.svg} – Units may choose to gain a Defense Token and end the Activation instead of attacking.
+Remember that Ranged Units \includesvg[height=10px]{\svgs/unit_ranged.svg} move after attacking, therefore they won't be able to move if they choose to Defend.
```

Comment:
I think it's unnecessary.

Reply:
You just wanted me to add the clarification about ranged units :)

Reply:
That was before you posted the picture about stacking abilities. It's unnecessary now.